docs/section10/10-14.md

```diff
@@ -0,0 +1,189 @@
+                    - Set expected_baseline_eff_b equal to 78% AFUE: `expected_baseline_eff_b = 0.78`
+                    - Check if the modeled efficiency meets the expected baseline efficiency by using the RCT compare_standard_val function, Permit_More_Stringent_Value is defined globally and is TRUE OR FALSE: `efficiency_matches_acceptable_value = compare_standard_val(Permit_More_Stringent_Value, expected_baseline_eff_b, modeled_efficiency_b, "greater than")`  func shall return True or False, "greater than" means that if the modeled_efficiency_b is greater than or equal to the expected_baseline_eff_b it would be accepted if Permit_More_Stringent_Value equals TRUE                  
+                - Reset was_efficiency_found equal to FALSE: `was_efficiency_found = FALSE`
+                - Loop through each of the efficiency_metric_types associated with the heating system until THERMAL_EFFICIENCY is found. Get the associated value of the efficiency from the list of efficiency_metric_values associated with the heating_system_b : `For x in range(len(heating_system_b.efficiency_metric_types)):`
+                    - Check if the efficiency metric equals THERMAL_EFFICIENCY: `if heating_system_b.efficiency_metric_types[x] == "THERMAL_EFFICIENCY":` 
+                        - Set the modeled efficiency value: `modeled_efficiency_b = heating_system_b.efficiency_metric_values[x]`
+                        - Set the efficiency found boolean variable to TRUE: `was_efficiency_found = TRUE`
+                    - Else, do nothing and move on to the next metric: `Else:`   
+                - Check if was_efficiency_found equals TRUE: `if was_efficiency_found == TRUE:`  
+                    - Set expected_baseline_eff_b equal to 80% Et: `expected_baseline_eff_b = 0.8`
+                    - Check if the modeled efficiency meets the expected baseline efficiency by using the RCT compare_standard_val function, Permit_More_Stringent_Value is defined globally and is TRUE OR FALSE: `efficiency_matches_acceptable_value = compare_standard_val(Permit_More_Stringent_Value, expected_baseline_eff_b, modeled_efficiency_b, "greater than")`  func shall return True or False, "greater than" means that if the modeled_efficiency_b is greater than or equal to the expected_baseline_eff_b it would be accepted if Permit_More_Stringent_Value equals TRUE                   
+            - Else, capacity is greater than or equal to 225,000 Btuh: `Else:`
+                - Loop through each of the efficiency_metric_types associated with the heating system until COMBUSTION_EFFICIENCY is found. Get the associated value of the efficiency from the list of efficiency_metric_values associated with the heating_system_b : `For x in range(len(heating_system_b.efficiency_metric_types)):`
+                    - Check if the efficiency metric equals THERMAL_EFFICIENCY: `if heating_system_b.efficiency_metric_types[x] == "COMBUSTION_EFFICIENCY":` 
```

Comment:
Send SG note about this: 78% per PNNL PRM RM Et whether or not Permit_More_Stringent_Value=True or FALSE

Reply:
Implemented but do we want to implement this for any other units. For example, in eQuest I do not believe you define COP at 47 and 17 so all eQuest projects would fail unless the vendors transcribed/mapped the EIR to those entries based ont he performance curves.

Reply:
Going to implement a special fail for if a project gets the high temperature efficiency correct and the low temperature efficiency is undefined.